implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverConfig.java

```diff
@@ -42,4 +45,8 @@ default String resourceType() {
         String resourceType = get(prefix() + ".resourceType");
         return resourceType == null ? "global" : resourceType;
     }
+
+    default Map<String, String> resourceLabels() {
+        return Collections.emptyMap();
```

Comment:
To allow for flexibility in implementations of the Config interfaces, we typically retrieve the config property using the `get` method which returns a String and then convert to the desired return type. For example, an implementation of `StackdriverConfig` could be `StackdriverConfig config = System::getProperty;` which allows configuring all the properties via system properties, except for resourceLabels right now.

I'm not sure how we would convert a String to a Map, though. We may not have much of a choice in this case.

Reply:
Yeah, I'm not sure there's a good way to handle that, since we'd have to define our own string-encoding of what a Map looks like. If you do want to go that route, let me know and I'll 

FWIW, I found another case where a Config object expects a map in `HumioConfig`. Looks like a nearly identical situation. There we're returning `null`, but I'd much rather return an empty map since it doesn't require any special handling.

https://github.com/micrometer-metrics/micrometer/blob/master/implementations/micrometer-registry-humio/src/main/java/io/micrometer/humio/HumioConfig.java#L48-L58